azext_edge/tests/edge/support/create_bundle_int/helpers.py

```diff
@@ -238,7 +240,9 @@ def get_file_map(
     mq_traces: bool = False,
 ) -> Dict[str, Dict[str, List[Dict[str, str]]]]:
     # Remove all files that will not be checked
-    arc_namespace, aio_namespace, acs_namespace, c_namespace = process_top_levels(walk_result, ops_service)
+    arc_namespace, aio_namespace, acs_namespace, ssc_namespace, c_namespace = process_top_levels(
```

Comment:
tbh this may need to be a dict at some point - tis too large for a tuple

Reply:
if you dont want to do this - put a todo and I will deal with it later

Reply:
updated, please feel free to take another look